projects/example-app/src/app/books/actions/book-exists-guard.actions.ts

```diff
@@ -0,0 +1,14 @@
+import { Action } from '@ngrx/store';
+import { Book } from '../models/book';
+
+export enum BookExistsGuardActionTypes {
+  Load = '[Book Exists Guard] Load',
```

Comment:
I think it would be better readable if the enum would be `LoadBook` and its value `[Book Exists Guard] Load Book`. What do you think?

Reply:
Yes, I agree. It's more specific this way and more readable. I will change as suggested.